build-configs/integreat-e2e/index.ts

```diff
@@ -5,14 +5,9 @@ import {
   WebBuildConfigType
 } from '../BuildConfigType'
 import integreatPlatformBuildConfigs from '../integreat'
+import { template } from '../integreat/assets/cityNotCooperatingTemplate'
```

Comment:
Change export to default and
```suggestion
import cityNotCooperatingTemplate from '../integreat/assets/cityNotCooperatingTemplate'
```